pages/analytics.vue

```diff
@@ -0,0 +1,15 @@
+<template>
+  <div class="flex flex-col pt-20">
+    <b class="text-center text-2xl">Clicks this month: {{ clickAnalytics }}</b>
```

Comment:
This should be formatted, but until then use something other than `<b>` for this. Something like:

```
<h1 class="font-bold text-xl mb-2">
  Clicks this month
</h1>
<pre>
  {{ clickAnalytics }}
</pre>
```